audio/pavucontrol/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                pavucontrol
+version             4.0
+license             GPL-2+
+categories          audio
+maintainers         {mit.edu:quentin @quentinmit} \
+                    openmaintainer
+platforms           darwin
+description         PulseAudio Volume Control
+
+long_description    PulseAudio Volume Control is a GTK+ volume control tool for PulseAudio.
+
+master_sites        https://www.freedesktop.org/software/pulseaudio/pavucontrol/
+homepage            ${master_sites}
+
+use_xz              yes
+
+checksums           rmd160  efb33c4f161e237b8da445fc9d6baff17be251dc \
+                    sha256  8fc45bac9722aefa6f022999cbb76242d143c31b314e2dbb38f034f4069d14e2 \
+                    size    165216
+
+depends_build-append port:pkgconfig \
+                     port:intltool
+
+depends_lib-append   port:portaudio \
+                     port:gtkmm3 \
+                     port:libsigcxx2 \
+                     port:libcanberra
+
+test.run            no
```

Comment:
now that is a bit unusual. We usually only say "test.run yes" or leave it out entirely, as the port system defaults to "no". I think you meant "yes" which is saying that that the port responds to "sudo port test".